### PR TITLE
Create separate debugging-doc wiki page for oppia-android

### PR DESCRIPTION
**Pull request by MohitGupta121**
## Explanation

Create separate debugging-doc wiki page for oppia-android

## Essential Checklist

- [x] The PR title and explanation each start with "Fix #bugnum: " (If this PR fixes part of an issue, prefix the title with "Fix part of #bugnum: ...".)
- [ ] Any changes to [scripts/assets](https://github.com/oppia/oppia-android/tree/develop/scripts/assets) files have their rationale included in the PR explanation.
- [x] The PR follows the [style guide](https://github.com/oppia/oppia-android/wiki/Coding-style-guide).
- [x] The PR does not contain any unnecessary code changes from Android Studio ([reference](https://github.com/oppia/oppia-android/wiki/Guidance-on-submitting-a-PR#undo-unnecessary-changes)).
- [x] The PR is made from a branch that's **not** called "develop" and is up-to-date with "develop".
- [x] The PR is **assigned** to the appropriate reviewers ([reference](https://github.com/oppia/oppia-android/wiki/Guidance-on-submitting-a-PR#clarification-regarding-assignees-and-reviewers-section)).

## For UI-specific PRs only

If your PR includes UI-related changes, then:
- Add screenshots for portrait/landscape for both a tablet & phone of the before & after UI changes
- For the screenshots above, include both English and pseudo-localized (RTL) screenshots (see [RTL guide](https://github.com/oppia/oppia-android/wiki/RTL-Guidelines))
- Add a video showing the full UX flow with a screen reader enabled (see [accessibility guide](https://github.com/oppia/oppia-android/wiki/Accessibility-A11y-Guide))
- Add a screenshot demonstrating that you ran affected Espresso tests locally & that they're passing

### CODE REVIEW

**Review by MohitGupta121 — COMMENTED** [review]
@seanlip @BenHenning PTAL, on the debugging docs wiki page.
Thanks!

File: wiki/Debugging.md [hunk 1]
```diff
@@ -9,4 +9,5 @@ Generally, finding the root cause of a bug is a question of strategically checki
 3. Based on the environment you've identified in steps 1 + 2, write a test case which the bug causes to fail (if one doesn't already exist). This gives you an automatic way to check whether or not the bug is fixed.
 4. Then, work towards the solution, until the test case you wrote in step 3 passes.
 
-Unless the bug you are trying to fix is trivial, we strongly recommend creating a debugging doc to organize your work. You can use this debugging doc to write down any pertinent information about the issue and to organize your debugging process, and it will make it much easier to ask for help from others since all the necessary information is there. You can find more guidance [here](https://github.com/oppia/oppia-android/wiki/Get-Help#before-you-ask-a-general-question) on how to write a debugging doc.
\ No newline at end of file
+Unless the bug you are trying to fix is trivial, we recommend creating a debugging doc to organize your work:
```
Comment (seanlip) [inline wiki/Debugging.md]: ... to organize your work and easily communicate it to other team members. See [Debugging Docs](...) for guidance on how to write a debugging doc.

File: wiki/Frequent-Errors-and-Solutions.md [hunk 2]
```diff
@@ -30,29 +30,4 @@ To solve this
...
-#### 4. Continue/Review from Mentor
-Keep going! Continue documenting your investigations, guesses, and tests of those guesses. You can share your debugging doc with your assigned onboarding mentor to review and help you in finding the root cause of the issue or the solution. 
-
-#### 5. Document Your Solution
-Once you figure out what the problem was, write that down too! Include an analysis of how the root cause you identify caused the errors you described at the top. Often, this will take the form of one of your suspected solutions.
-
-#### Get Started
-Ready to get started with your own debugging doc? You can make a copy of [this template](https://docs.google.com/document/d/1OBAio60bchrNCpIrPBY2ResjeR11ekcN0w5kNJ0DHw8/edit?usp=sharing) to get started. 
\ No newline at end of file
+Follow the guidelines outlined on this [wiki page](https://github.com/oppia/oppia-android/wiki/Debugging).
```
Comment (seanlip) [inline wiki/Frequent-Errors-and-Solutions.md]: If you are running into an error that you cannot debug, follow the ...

File: wiki/Get-Help.md [hunk 3]
```diff
@@ -100,9 +100,9 @@ paste a screenshot
 
 ### Before you ask a general question
 
-* We expect that you have already **set up Oppia on your machine**, and it is successfully running. (If not, kindly do that first!)
-* Prepare a debugging doc following [this template](https://docs.google.com/document/d/1OBAio60bchrNCpIrPBY2ResjeR11ekcN0w5kNJ0DHw8). For more information on how to prepare a debugging doc, follow the guidelines provided on [this wiki page](https://github.com/oppia/oppia-android/wiki/Frequent-Errors-and-Solutions#how-to-write-a-debugging-doc).
-* If there are **checks** on your PR, and you haven’t done any changes in that direction, kindly understand that sometimes they just fail due to flakiness. You should request for a re-run of those only when it’s preventing your PR from getting merged.
+- We expect that you have already **set up Oppia on your machine**, and it is successfully running. (If not, kindly do that first!)
+- Prepare a debugging doc following [the guidelines provided on the wiki page](https://github.com/oppia/oppia-android/wiki/Debugging-Docs).
```
Comment (seanlip) [inline wiki/Get-Help.md]: The "sentence reading" of this is a bit weird, in terms of what gets linked. Perhaps "the guidelines provided on the [Debugging Docs wiki page](...)".

File: wiki/Get-Help.md [hunk 4]
```diff
@@ -100,9 +100,9 @@ paste a screenshot
...
+- If there are **checks** on your PR, and you haven’t done any changes in that direction, kindly understand that sometimes they just fail due to flakiness. You should request for a re-run of those only when it’s preventing your PR from getting merged.
```
Comment (seanlip) [inline wiki/Get-Help.md]: I don't actually understand what this means. Perhaps:

If there are **failing checks** on your PR, and you haven't done any changes that might cause them, kindly ...

**Review by seanlip — CHANGES_REQUESTED** [review]
Thanks @MohitGupta121, left some comments.

File: wiki/Debugging.md (hunk 1 above)
Comment (MohitGupta121) [reply wiki/Debugging.md]: Done.

File: wiki/Frequent-Errors-and-Solutions.md (hunk 2 above)
Comment (MohitGupta121) [reply wiki/Frequent-Errors-and-Solutions.md]: Done.

File: wiki/Get-Help.md (hunk 3 above)
Comment (MohitGupta121) [reply wiki/Get-Help.md]: Done.

File: wiki/Get-Help.md (hunk 4 above)
Comment (MohitGupta121) [reply wiki/Get-Help.md]: Done.

**Review by MohitGupta121 — COMMENTED** [review]
Thanks @seanlip, PTAL, make changes as per suggested.

**Review by seanlip — APPROVED** [review]
Great, thanks @MohitGupta121! LGTM.

**Pull request merged**